README.md

```diff
@@ -144,10 +145,14 @@ Second, you can set the values via the configuration object:
 config.solid_errors.send_emails = true
 config.solid_errors.email_from = "errors@myapp.com"
 config.solid_errors.email_to = "devs@myapp.com"
+# Tell Solid Errors whether or not to limit the total emails per occurrence. Defaults to false.
+config.solid_errors.one_email_per_occurrence = true
 ```
 
 If you have set `send_emails` to `true` and have set an `email_to` address, Solid Errors will send an email notification whenever an error occurs. If you have not set `send_emails` to `true` or have not set an `email_to` address, Solid Errors will not send any email notifications.
 
+If you have set `send_emails` to `true` and have set `one_email_per_occurrence` to `true`, Solid Errors will only send an email notification after the first occurrence of a new error or the first reoccurrence of a previously resolved error. If you have not set `one_email_per_occurrence` to `true`, Solid Errors will send an email notification for each occurrence of an error.
```

Comment:
Should this simply be the default behavior and we remove the configuration altogether? Can you think of any reason someone would _want_ to have every occurrence of every error send an email?

Reply:
Gosh, I really cannot think of a reason. I know I have enough "noise" in my inbox, message apps, etc. that I prefer fewer emails (in general LOL).

Reply:
Me too. Let's remove the configuration option and just make this how the gem behaves